cleaning/pdf-mining/pdftotxt.py

```diff
@@ -18,8 +19,7 @@ def parse_statement_date(line, iterator):
     line_lower = line.lower()
     for clue in DATE_CLUES:
         if clue in line_lower:
-            statement_date = dateparser.parse(line_lower.split(clue)[-1],
-                                              languages=LANGUAGES)
```

Comment:
is this related to language detection?

Reply:
where the `dateparser` come from? https://github.com/scrapinghub/dateparser? is this a global thing?

Reply:
It's a python package. It's in [requirements.txt](https://github.com/weilu/cs4225/blob/money/requirements.txt#L14) and imported at the beginning of this file.